.github/workflows/ci.yml

```
@@ -15,13 +15,42 @@ concurrency:
 
 jobs:
 
+#----------------------------------------------------------------------------
+# Detect checkboxes are checked
+#----------------------------------------------------------------------------
+  detect_checkboxes:
+    name: Detect Checkboxes
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+    outputs:
+      checked: ${{ steps.detect.outputs.checked }}
+      unchecked: ${{ steps.detect.outputs.unchecked }}
+
+    permissions:
+      pull-requests: write
+
+    steps:
+      - name: Checkbox Trigger
+        id: detect
+        uses: a7med3bdulbaset/checklist@3
```

Looks like the repository is now:
```suggestion
        uses: AhmedBaset/checklist@3
```